### ISSUE TITLE
Transfer settings from one machine to another

### ISSUE THREAD

**Issue by SOHODeveloper** [Microsoft/VS-PPT#147]
It appears that the standard export/import of VS settings does not transfer any of the Custom Document Well settings.  I have done extensive color changes to the tab colors via RegEx. 
-  Please somehow provide a mechanism for transferring the settings
- As immediate help, please share where the setting for the 2017 version are so that I can transfer them manually.

**Comment by yepeekai** [comment]
Also, visual studio 2017 is not perfectly stable (I had to delete the local folder 4 times).  So it would be really useful to restore the settings for the custom document well when that happens

**Comment by justcla** [comment]
I agree it would be useful to export/import CDW settings (and settings related to other extensions too).
We don't currently have a way to do this. I will see what I can learn about where the settings are stored.

**Comment by SOHODeveloper** [comment]
Hi justcla,
Any progress on determining where the settings are stored?
Thanks
